<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main_window</class>
 <widget class="QMainWindow" name="frm_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Task Manager Beta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="wg_main_tasks" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_tasks">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Aufgaben</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBox" name="tb_tasks">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="work">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>98</width>
            <height>76</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Arbeit</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="lw_work">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="home">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>252</width>
            <height>238</height>
           </rect>
          </property>
          <attribute name="label">
           <string>zu Hause</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="lw_home"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="other">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>98</width>
            <height>76</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Sonstige</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QListWidget" name="lw_other"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="finished_tasks">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>98</width>
            <height>76</height>
           </rect>
          </property>
          <attribute name="label">
           <string>erledigte Aufgaben</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListWidget" name="lw_finished"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wg_tasks_buttons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_new_task">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Neue Aufgabe</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_show_description">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Detailansicht</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wg_main_description" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_description">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Beschreibung</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_task_title">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ãœberschrift Aufgabe</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="te_description">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wg_description_time" native="true">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QDateEdit" name="ded_description">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
            <property name="date">
             <date>
              <year>2025</year>
              <month>6</month>
              <day>19</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="ted_description">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="time">
             <time>
              <hour>10</hour>
              <minute>20</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wg_description_buttons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_finish_task">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>AbschlieÃŸen</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_edit_task">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Bearbeiten</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
